src/util/saveArticles.ts

```diff
@@ -112,6 +116,7 @@ async function downloadBulk(listOfArguments: any[], downloader: Downloader): Pro
 
                 return downloader.downloadContent(arg.val.url).then((r) => {
                     resp.result = r;
+                    resp.path += resp.result.responseHeaders.path_postfix ? resp.result.responseHeaders.path_postfix : '';
```

Comment:
`resp.path += resp.result.responseHeaders.path_postfix || ''` would be better I believe